### PR TITLE
Update dependencies and travis for node 6

### PR DESCRIPTION
**Pull request by geek**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,5 +4,6 @@ node_js:
   - 4.0
   - 4
   - 5
+  - 6
```
Comment (hueniverse) [inline .travis.yml]: We should only be testing "4" and "node".

Comment (ruiquelhas) [inline .travis.yml]: @hueniverse is there any organization "policy" for this? It looks like we are testing v5 on some modules.

Comment (arb) [inline .travis.yml]: I'm confused, over [here](https://github.com/hapijs/contrib/issues/80#issuecomment-217355798) you said 
> "4" and "6" 
but here you are saying "4" and "node"?

Comment (Marsup) [inline .travis.yml]: "node" === "6" currently.
From nvm docs : `node: this installs the latest version of node`

**Comment by ruiquelhas** [comment]
Looks like most modules in the org are moving to this `.travis.yml` spec, so LGTM.

**Pull request merged**